test/Microsoft.ML.Tests/Microsoft.ML.Tests.csproj

```diff
@@ -72,8 +72,8 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetArchitecture)' == 'x64'">
-    <PackageReference Include="libtorch-cpu-win-x64" Version="$(LibTorchVersion)" Condition="$([MSBuild]::IsOSPlatform('Windows')) AND '$(TargetArchitecture)' == 'x64'" />
-    <!-- <PackageReference Include="TorchSharp-cuda-windows" Version="0.96.8" Condition="$([MSBuild]::IsOSPlatform('Windows'))" /> -->
+    <!--<PackageReference Include="libtorch-cpu-win-x64" Version="$(LibTorchVersion)" Condition="$([MSBuild]::IsOSPlatform('Windows')) AND '$(TargetArchitecture)' == 'x64'" />-->
+     <PackageReference Include="TorchSharp-cuda-windows" Version="0.96.8" Condition="$([MSBuild]::IsOSPlatform('Windows'))" /> 
```

Comment:
Do our build machines have cuda GPUs or should this be reverted before checked in?

Reply:
Can  we update to [0.99.3](https://www.nuget.org/packages/TorchSharp-cuda-windows/0.99.3)

Reply:
Or should we ...

Reply:
So it should be reverted, but it also wouldn't cause any problems since the GPU package also has the CPU code, and if it can't detect the GPU it just does CPU. That being said, its much larger and takes longer, so not good for CI.

Reply:
And I'll do the torch sharp update in another PR. I would want to do some testing with the new version before I just push that change.